ganttproject-tester/test/net/sourceforge/ganttproject/chart/mouse/ChangeTaskProgressRulerTest.java

```diff
@@ -49,6 +49,32 @@ private static Task createTask(TaskManager taskManager) {
         return result;
     }
 
+    public void testProgressConversionToUnits() {
+        TaskChartModelFacade mockChartModel = EasyMock.createMock(TaskChartModelFacade.class);
```

Comment:
extract this code which creates a task, rectangles and ruler into a helper method? It is duplicated in two tests.